rust-toolchain.toml

```
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "1.81.0"
+profile = "minimal"
+components = [ "rustfmt", "clippy" ]
+
```

We should add the windows (?) and wasm targets here (which are added to the current host target thankfully).

---

Ignore the windows part, we already have `targets = ["x86_64-pc-windows-gnu"]` in the rust-toolchain.toml for windowsauth, so all we need here is something like

```suggestion
targets = [ "wasm32-unknown-unknown" ]
```